ui/app/AppLayouts/Wallet/panels/DAppsWorkflow.qml

```diff
@@ -151,8 +155,10 @@ DappsComboBox {
                 function onMaxFeesUpdated(maxFees, maxFeesWei, haveEnoughFunds, symbol) {
                     maxFeesText = `${maxFees.toFixed(2)} ${symbol}`
```

Comment:
This should be formatted using `LocaleUtils`; kinda hard to do with the `ethStr` below... :thinking:

Reply:
This is just a dummy display code to show that the data is coming from the SDK. It has to be replaced with the proper status-go based implementation here in this work package: https://github.com/status-im/status-desktop/issues/15192

Reply:
Ah cool